app/f0_serialcom/blinky.h

```diff
@@ -0,0 +1,5 @@
+#ifndef _BLINKY_H_
+#define _BLINKY_H_
+
+void blinky_task(void *params);
+#endif	/* BLINKY_H */
```

Comment:
Have an empty line between the function declarations and the endif, maybe? /bikeshed-bikeshed-bikeshed :)